### PR TITLE
Fix elimination of module-info.java javadoc in openj9 docs

### PR DESCRIPTION
**Pull request by lumpfish**
Signed-off-by: Simon Rushton <rushton@uk.ibm.com>

### CODE REVIEW

**Comment by lumpfish** [comment]
Jenkins compile win jdk11

File: closed/custom/Docs.gmk [hunk 1]
```diff
@@ -191,21 +191,20 @@ OPENJ9_ONLY_JAVADOC_JAVA_ARGS = -Djava.awt.headless=true -Dextlink.spec.version=
...
     <div class="block"> \
       <dl class="notes"> \
         <dd>$(OPENJDK_MODULE_REFERENCE_TEXT)</dd> \
       </dl> \
     </div> \
-  </section>
-
+  </section> \
+  \$(NEWLINE)<!-- ============ PACKAGES SUMMARY =========== -->
+
```
Comment (keithc-ca) [inline closed/custom/Docs.gmk:207]: Please remove trailing whitespace.

File: closed/custom/Docs.gmk [hunk 2]
```diff
@@ -259,9 +253,7 @@ OPENJ9_ONLY_JAVADOC_SCRIPT_BODY = \
       regex="$$topdir/.*$$excepttopdir" ; \
       if ! $(GREP) -qe "$$regex" $(OPENJ9_ONLY_JAVADOC_FILES) ; then \
```
Comment (keithc-ca) [inline closed/custom/Docs.gmk:254]: What is the point of this use of `grep`? The files named in `OPENJ9_ONLY_JAVADOC_FILES` all match `*.java` and the files we're considering here are named `module-summary.html`.

Comment (lumpfish) [reply closed/custom/Docs.gmk:254]: The code is looking for a module-info.java file provided by openj9 - in that case we want to leave the generated module-summary.html unchanged.
So for example:
1. When the regex is  openj9.cuda/.*module-info.java a file is found and the module-summary.html is left unchanged.
2. When the regex is  jdk.management/.*module-info.java no file is found and the module-summary.html is edited.

File: closed/custom/Docs.gmk (hunk 1 above)
Comment (lumpfish) [reply closed/custom/Docs.gmk]: Removed the space in the line consisting of a single space (line 207).

**Comment by keithc-ca** [comment]
Please squash.

**Comment by keithc-ca** [comment]
Why was this closed? Is the intention that what was removed by this will not be removed in jdk15 or jdknext?

**Comment by lumpfish** [comment]
I thought this had been merged and I deleted my branch.
I've recreated my branch and submitted another PR: https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/334

**Pull request closed without merging**